BDArmory/Distribution/GameData/BDArmory/ChangeLog.txt

```diff
@@ -20,6 +20,12 @@
 			- IMPORTANT: "LoftTermRange" has been renamed to "terminalHomingRange"!
 		- Fixed MultiMissileLauncher behavior with loft guidance, as well as pronav/augpronav.
 		- Added new "aamhybrid" homingType, uses lead guidance until "terminalHomingRange" is reached, then it will switch to "terminalHomingType", with the same options as loft guidance.
+        - new 'hasDataLink' field that can be used to Launch Radar/Heat missiles and guide them later with help of an external radar or out of range of your radar/seeker an re-acquire a lock when inside radar/seeker range.
+		    - Datalink Can also guide missiles outside they Max offboresight, but with a precision lost until the target get inside the boresight again
```

Comment:
This is more a of a Complement for aamloft since if the missile had a small maxoffboresight the loft didn't happen, with datalink now it bypass the maxoffboresight limit but as you saw on missilelauncher generates a drift